libs/pages/cluster/src/lib/ui/page-settings-features/page-settings-features.tsx

```diff
@@ -12,19 +12,27 @@ import GcpExistingVPC from './gcp-existing-vpc/gcp-existing-vpc'
 
 export interface PageSettingsFeaturesProps {
   loading: boolean
-  features?: ClusterFeature[]
+  features?: ClusterFeatureResponse[]
   cloudProvider?: CloudProviderEnum
 }
 
 export function PageSettingsFeatures(props: PageSettingsFeaturesProps) {
   const { loading, features, cloudProvider } = props
 
   const featureExistingVpc = features?.find(({ id }) => id === 'EXISTING_VPC')
-  const featureExistingVpcValue = featureExistingVpc?.value
+  const featureExistingVpcValue = featureExistingVpc?.value_object
 
   const featureExistingVpcContent = match(cloudProvider)
-    .with('AWS', () => <AWSExistingVPC feature={featureExistingVpcValue as ClusterFeatureAwsExistingVpc} />)
-    .with('GCP', () => <GcpExistingVPC feature={featureExistingVpcValue as ClusterFeatureGcpExistingVpc} />)
+    .with('AWS', () =>
```

Comment:
Can you `match` on `featureExistingVpcValue?.type` directly ?

Reply:
yep https://github.com/Qovery/console/pull/1388/commits/04a39ebd5f56d9546d27c0f28d2b128b95f7888f